stable/pipelines/templates/ml-pipeline/persistenceagent/role-binding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: {{ template "persistenceagent.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "pipelines.name" . }}
```

Comment:
I believe the subcomponents of pipelines (e.g. `persistenceagent`) had different app labels originally.
Please make sure you are consitent with this - here the label should probably be:
```
app: {{ template "persistenceagent.name" . }}
```
And you're making sure that by default this is prefixed with release name